rpctest/node.go

```diff
@@ -117,10 +107,7 @@ func (n *nodeConfig) arguments() []string {
 	args = append(args, fmt.Sprintf("--rpccert=%s", n.certFile))
 	// --rpckey
 	args = append(args, fmt.Sprintf("--rpckey=%s", n.keyFile))
-	if n.dataDir != "" {
-		// --datadir
-		args = append(args, fmt.Sprintf("--datadir=%s", n.dataDir))
-	}
+	args = append(args, fmt.Sprintf("--appdata=%s", n.appDataDir))
 	if n.logDir != "" {
```

Comment:
logDir is relative to the appdata dir, we don't need this arg.  Remove this along with the (now unused) dataDir and logDir fields in the node config struct.